tobiasHe/assignments/treesBFS/lc637/lc637.js

```diff
@@ -0,0 +1,49 @@
+// Problem Statement #
+
+// Given a binary tree, populate an array to represent the averages of all of its levels.
+
+class TreeNode {
+	constructor(value) {
+		this.value = value;
+		this.left = null;
+		this.right = null;
+	}
+}
+/* 
+  using BFS to traverse the tree, while keeping a sum value of each level
+  initialize a queue with root
+  keep track of queueLength, to move on to next levels
+  shift out the queue and check if current.value has any left and right children
+  if there are any, add them to the queue 
+  then push the average of that level into result
+  return result 
+
```

Comment:
Nice overview - but missing some steps like keeping track of the sum of the nodes in the level. Could also include some details about edge cases here.